doc/wiki/Contribute:-Update-Wiki.md

```diff
@@ -1 +1,115 @@
+
+## Setup
+**_This setup only needs to be done once (the first time) !!!_**
+
+1. **Check GAMER version**
+   * Please make sure you have the latest GAMER.
+
+1. **Create the first page**
+   * Click the Wiki page and create the first page
+
+   [[images/CreateWiki.png]]
+
+1. **Create a token for the action bot**
+   * Go to `Setting` of your account > `Developer setting` > `Personal access tokens` > `Generate new token`
+      - NOTE: We use the `Tokens(classic)` in this example.
+   * Please check the `repo` and the `workflow` options.
+      - NOTE: You can set the `Note` of this token freely
+   * You might want to set the `Expiration` to `No expiration`.
+   * Click the green `Generate token` at the bottom.
+
+   [[images/CreateToken.png]]
+
+   * Remember to save the token since it will only be shown once!
+
+1. **Create the repository secret token and email**
+   * Go to `Setting` of your forked gamer repository > `Security` > `Secrets and variables` > `Actions` > `Repository secrets`
+   
+   [[images/CreateSecret.png]]
+
+   * Click `New repository secrets`, and then you will see the following. Please replace `<your_personal_access_token>` with the token generated in the previous step at `Secret` and make sure the `Name` of the secret is `TOKEN_FOR_WIKI`. 
+
+   [[images/SetToken.png]]
+
+   * Click `New repository secrets`. Please replace `<your_email_address>` with your account email address at `Secret` and make sure the `Name` of the secret is `MY_EMAIL`.
+      - NOTE: We will not know your address since it only exists in your repository. This step is only for recording your contribution by the action.
+
+   [[images/SetMail.png]]
+
+1. **Enable actions (workflows)**
+   * Click `Actions` > click the green button.
+   ![image](https://github.com/ChunYen-Chen/CheckNode/assets/70311975/9e58d4a8-3248-4ceb-81ff-276a6943149d)
+
+1. **Initialize wiki**
+   * Click `Action` > `Workflows` > `Copy doc to wiki` > `Run workflow` > choose `Branch: main` > Click green `Run workflow`. Once the workflow is done, the wiki is also updated to the `main` branch version.
+
+   [[images/InitializeWiki.png]]
+
+## Edit Wiki
+We provide three examples of editing the wiki pages: through `gollum` (recommended), from the GitHub website, and on the local terminal. In the following examples, we call the new branch you would like to contribute as `new_contribution_branch`.
+
+1. **Through `gollum` (recommended)**
+   - Install [gollum](https://github.com/gollum/gollum).
+   - Click `Action` > `Workflows` > `Copy doc to wiki` > `Run workflow` > choose `new_contribution_branch` branch > Click green `Run workflow`. Once the workflow is done, Wiki is also updated to the `new_contribution_branch` branch version.
+   - Clone your forked Wiki git. You may find the Wiki URL at the bottom right of the Wiki page.
+
+     [[images/WikiGitLocation.png]]
+
+   - Edit by `gollum`. 
+     * `gollum /path/to/wiki`
```

Comment:
Using `gollum --lenient-tag-lookup /path/to/wiki` resolves the issue of some files in subdirectories not linking correctly.
[gollum wiki](https://github.com/gollum/gollum/wiki#page-sanitization:~:text=%2D%2Dlenient%2Dtag%2Dlookup)

Reply:
Thanks a lot! I have updated it.